src/raw/baby-onesie/locales/en/messages.po

```diff
@@ -19,3 +19,4 @@ msgstr ""
 #: scripts/temp/baby-onesie.js:2
 msgid "icon.title.baby-onesie"
 msgstr "Baby onesie"
+
```

Comment:
This is a test to see if this comment is forwarded to an English translator in Crowdin :)

Reply:
This is test to see if this comment is forwarded to an English translator in Crowdin :)

Reply:
This is another test to see if this comment is forwarded to an English translator in Crowdin :)